tutorial/gui/pcl-visualizer/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+project(tutorial-grabber)
+
+cmake_minimum_required(VERSION 3.0)
+
+find_package(VISP REQUIRED visp_core visp_gui)
+
+# set the list of source files
+set(tutorial_cpp
+  tutorial-pcl-visualizer.cpp
+)
+
+# set(tutorial_data "${CMAKE_CURRENT_SOURCE_DIR}/video.mp4")
+
```

Comment:
Remove useless lines that are commented

Reply:
Done in [91391b3](https://github.com/lagadic/visp/pull/1163/commits/91391b324ed48da41cfb5034f2807dc2e77fa549)